eof.c

```diff
@@ -0,0 +1,36 @@
+#include "shell.h"
+
```

Comment:
**Nice help!**
Before merge, I would need you to comment the code to pass betty style checks.
Then I can resolve the conflicts for your commits!

Hope you being well, remember we have until tonight to test the code!